### PR TITLE
Add support for hash: ignore

### PR DESCRIPTION
**Pull request by jmelis**

### CODE REVIEW

**Comment by jmelis** [comment]
@riuvshin @jfchevrette can you review?

I marked it as WIP because I still want to add some tests for this, but the functionality should be ready.

File: saasherder/saasherder.py [hunk 1]
```diff
@@ -152,13 +152,44 @@ def mkdir_templates_dir(self):
         if not os.path.isdir(self.templates_dir):
             os.mkdir(self.templates_dir)
 
+    def resolve_download_hash(self, service_hash):
+        """
+        Resolve download hashes with special values
+        """
+
+        if service_hash == 'ignore':
+            return 'master'
```
Comment (riuvshin) [inline saasherder/saasherder.py:161]: does this works for both GL and GH?

**Review by riuvshin — APPROVED** [review]
looks good!

**Comment by jmelis** [comment]
@jfchevrette this is ready for merge, but can you please review first?

**Comment by riuvshin** [comment]
@jmelis can we have at least a unit test that will test `raw_gitlab()` ? we can't have e2e becuase of vpn but maybe we can mock it

**Comment by jmelis** [comment]
@riuvshin that is a good idea, but it would take some effort to do so

**Comment by jfchevrette** [comment]
LGTM! Thanks a lot @jmelis

**Pull request merged**